tests/network/vmi_multus.go

```diff
@@ -186,13 +186,13 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		Expect(createPtpNetworkAttachmentDefinition(tests.NamespaceTestAlternative, ptpConf2, ptpSubnet)).To(Succeed())
 
 		// Multus tests need to ensure that old VMIs are gone
-		Eventually(func() int {
+		Eventually(func() []v1.VirtualMachineInstance {
```

Comment:
I'm tempted to consider this a premature optimization at best, since we (at least I don't...) have no clue about how much it costs by copying the values, rather than passing the pointers.

It a lot more comfortable to pass the full structs, since I can just `return append(list1.Items, list2.Items...)` whereas I'd have to iterate each of the lists and append the pointers of each of the entries to the slice. 

I'll do it if you insist, but I think the code will look clunkier.

Reply:
Hmm, the readability of `return append(list1.Items, list2.Items...)` is indeed better, that's a fair point.
I'm convinced :+1:

Reply:
Thanks @rhrazdil . If this were production code, I think you would be right: the performance should be optimized.

Since this is test code, I think the overall test readability / simplicity should be preferred over performance.